packages/core/src/plugins/use-error-handler.ts

```diff
@@ -1,7 +1,15 @@
 import { Plugin, DefaultContext, TypedExecutionArgs, ExecutionResult } from '@envelop/types';
 import { handleStreamOrSingleExecutionResult } from '../utils.js';
 
-export type ErrorHandler = (errors: readonly Error[] | any[], context: Readonly<DefaultContext>) => void;
+export type ErrorHandler = ({
+  errors,
+  context,
+  phase,
+}: {
+  errors: readonly Error[] | any[];
```

Comment:
Can we remove the any and use the same GraphQL Error type we used in the masked error plugin?